### ISSUE TITLE
Check if `string.strip_tags `, `mcrypt.*`, and `mdecrypt.*` filters are still available in PHP 8

### ISSUE THREAD

**Issue by Girgias** [php/doc-en#3860]
Going through our filter documentation and saw the deprecation warning boxes.

If those are still available we need to add an issue on php-src to remove them in PHP 9, if not update the docs.

From manual page: https://php.net/filters.encryption

---

**Comment by cmb69** [comment]
mcrypt.* and mdecrypt.* are no longer php-src concern, since ext/mcrypt has been unbundled. Not sure what to do with the documentation; https://pecl.php.net/package/mcrypt says no longer maintained, but had a couple of releases, the last one not even a year ago.

string.strip_tags is removed as of PHP 8.0.0 (https://heap.space/search?project=PHP-7.4&project=PHP-8.0&full=&defs=strfilter_strip_tags_create&refs=&path=&hist=&type=c).

**Comment by Girgias** [comment]
We probably should say that it was removed from PHP in PHP 7.2.0, as this is when we removed it. We could add a secondary note that it is still available on PECL.

**Comment by cmb69** [comment]
Yeah, something like https://www.php.net/manual/en/intro.mcrypt.php
